### PR TITLE
F upgrade storybook migrate dp input

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpInput stories to v7.

### CODE REVIEW

File: src/components/DpInput/DpInput.stories.mdx [hunk 1]
```diff
@@ -76,7 +76,7 @@ If an input field is `required`, an asterisk will be added to its label.
             label: {
                 text: 'Phone'
             },
-            pattern: '^(\+?)(-| |[0-9]|\(|\))*$',
+            pattern: '^(\\+?)(-| |[0-9]|\\(|\\))*$',
```
Comment (hwiem) [inline src/components/DpInput/DpInput.stories.mdx:79]: Why these changes?

Comment (ahmad-demos) [reply src/components/DpInput/DpInput.stories.mdx:79]: I hava this error msg `Unable to check <input pattern='^(+?)(-| |[0-9]|(|))*$'> because the pattern is not a valid regexp: nothing to repeat`

**Review by hwiem — APPROVED** [review]
lgtm
